### Question 1

In [12]:
q1Data = read.csv("Quiz3_DataSet1_3.csv", colClasses = c('factor', 'numeric'))
t.test(Sugar~Generation, data=q1Data)


	Welch Two Sample t-test

data:  Sugar by Generation
t = -4.5735, df = 195.76, p-value = 8.496e-06
alternative hypothesis: true difference in means between group Adult and group Teenager is not equal to 0
95 percent confidence interval:
 -29.04395 -11.54249
sample estimates:
   mean in group Adult mean in group Teenager 
              88.44919              108.74241 


mean difference : 108.742 - 88.449 = 20.29

assuming two - tailed (no directionality):
qt(0.025, 195.76)

### Question 2

In [13]:
abs(qt(0.025, 195.76))

[1] 1.972156

In [14]:
q2Data = read.csv("Quiz3_DataSet2_11.csv", colClasses=c('factor', 'factor'))
chisq.test(q2Data$Sugar, q2Data$Sex)


	Pearson's Chi-squared test

data:  q2Data$Sugar and q2Data$Sex
X-squared = 1.5332, df = 2, p-value = 0.4646


In [15]:
qchisq(0.95, 2) # it's a 1 tailed test with df = 2, a = 0.05

[1] 5.991465